plugins/log_full.yml

```diff
@@ -53,3 +53,13 @@ plugin:
     - $NAMESPACE
     - --context
     - $CONTEXT
+  download-container-logs:
+    shortCut: Ctrl-D
+    description: "download full log"
```

Comment:
I think it would be good to indicate to the users where the logs will be downloaded.